src/index.js

```diff
@@ -215,13 +222,17 @@ class Explosion extends React.PureComponent<Props, State> {
           });
           const containerTransform = [{translateX: left}, {translateY: top}];
           const transform = [{rotateX}, {rotateY}, {rotate: rotateZ}, {translateX}];
+          if (Platform.OS === 'android') {
+            transform.push({ perspective: 100 });
+          }
 
           return (
             <Confetti
               color={item.color}
               containerTransform={containerTransform}
               transform={transform}
               opacity={opacity}
+              renderToHardwareTextureAndroid={renderToHardwareTextureAndroid && isAnimating}
```

Comment:
so the confetti will be re-rendered every time we start or stop the animation, the idea of using native animation is to not re-render them (not use a local state so).

don't you think we could avoid `isAnimating` state and set statically `renderToHardwareTextureAndroid` on `AnimatedView` (`Confetti` component), even when the animation is stopped ?

Reply:
So I made an assumption based on the documentation this would be needed to reduce memory usage, since it states on the react native site:

> The downside is that this can use up limited video memory, so this prop should be set back to false at the end of the interaction/animation.

However after doing some profiling with our application + using this branch, I wasn't able to find any difference in energy or memory usage when this was always set to `true`.

Great catch, glad you asked!  I've removed all the changes to only assign this to true while animating based on state.